src/workers.rs

```diff
@@ -4,14 +4,15 @@ use redis;
 use redis::Commands;
 use cmdline_words_parser::StrExt;
 use std::process::Child;
+use errors::*;
 
 pub trait Worker {
     fn app(&self) -> &String;
     fn path(&self) -> &String;
     fn command(&self) -> &String;
     fn kind(&self) -> &str;
-    fn work_to_do(&self, &redis::Connection) -> bool;
-    fn work_being_done(&self, &redis::Connection) -> bool;
+    fn work_to_do(&self, &redis::Connection) -> Result<bool, FromanError>;
+    fn work_being_done(&self, &redis::Connection) -> Result<bool, FromanError>;
```

Comment:
could [do this](https://doc.rust-lang.org/book/first-edition/error-handling.html#the-result-type-alias-idiom) everywhere

```rust
type FromanResult<T> = result::Result<T, FromanError>;

// elsewhere

fn work_being_done(&self, &redis::Connection) -> FromanResult<bool>;
```

Reply:
Oooo, that's nice. Thanks!